packages/core/src/nativeModulesTypes.ts

```diff
@@ -24,22 +25,6 @@ export type DdNativeLogsType = NativeDdLogs;
  */
 export type DdNativeTraceType = NativeDdTrace;
 
-/**
- * A configuration object to initialize Datadog's features.
```

Comment:
We were basically duplicating this type here and on `sdk/types.tsx`, so I decided to unify them into one to avoid potential discrepancies.